test/thirdparty.test.ts

```diff
@@ -0,0 +1,227 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+import { deepStrictEqual, strictEqual } from "assert";
+import { describe, test } from "mocha";
+import { getAccounts } from "@tableland/local";
+import {
+  D1Orm,
+  DataTypes,
+  Model,
+  GenerateQuery,
+  QueryType,
+  type Infer,
+} from "d1-orm";
+import sql, { FormatConfig } from "@databases/sql";
+import { escapeSQLiteIdentifier } from "@databases/escape-identifier";
+import { NonceManager } from "@ethersproject/experimental";
+import { getDefaultProvider, overrideDefaults } from "../src/helpers/index.js";
+import { Database } from "../src/index.js";
+
+overrideDefaults("localhost", {
+  contractAddress: "0xe7f1725E7734CE288F8367e1Bb143E90bb3F0512",
+});
+
+describe("thirdparty", function () {
+  // Note that we're using the second account here
+  const [, wallet] = getAccounts();
+  const provider = getDefaultProvider("http://127.0.0.1:8545");
+  // const signer = wallet.connect(provider);
+  const baseSigner = wallet.connect(provider);
+  // Also demonstrates the nonce manager usage
```

Comment:
Yeh, pretty cool. This is showcasing how someone might want to use a nonce-tracker with tableland. It is as easy as wrapping your signer with a NonceManager... nice!